### PR TITLE
run oom on komodo

### PR DESCRIPTION
**Pull request by eivindjahren**
**Issue**
Resolves https://github.com/equinor/komodo-releases/issues/5858

**Approach**
_Short description of the approach_

(Screenshot of new behavior in GUI if applicable)

- [ ] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Added appropriate release note label
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Make sure tests pass locally (after every commit!)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

**Comment by berland** [comment]
Delete the lines in test_cli.py that you have copied from?

File: tests/integration_tests/scheduler/test_oom.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        ),
+        encoding="utf-8",
+    )
+    os.chmod("memory_hog.sh", 0o0755)
+    with pytest.raises(
+        ErtCliError,
+        match="Realization.*0 failed.*memory_hog.*was killed due to out-of-memory",
+    ):
+        run_cli(
+            "test_run",
```
[the remark this answers is not in the document]
Comment (eivindjahren) [reply tests/integration_tests/scheduler/test_oom.py:56]: Yes

[the remark this answers is not in the document]
Comment (eivindjahren) [reply tests/integration_tests/scheduler/test_oom.py]: Done now.

File: ci/testkomodo.sh [hunk 2]
```diff
@@ -51,7 +51,7 @@ start_tests () {
     pushd ${CI_TEST_ROOT}/tests
 
     python -m pytest -n auto --mpl --benchmark-disable --eclipse-simulator \
-        --durations=0 -sv --dist loadgroup -m "not limit_memory" --max-worker-restart 0
+        --durations=0 --runoom -sv --dist loadgroup -m "not limit_memory" --max-worker-restart 0
```
[the remark this answers is not in the document]
Comment (eivindjahren) [reply ci/testkomodo.sh:54]: Yes, my bad

File: tests/integration_tests/scheduler/test_oom.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+from textwrap import dedent
+
+import pytest
+
+from ert.cli.main import ErtCliError
+from tests.integration_tests.run_cli import run_cli
+
+
+@pytest.fixture()
+def queue_name_config():
```
[the remark this answers is not in the document]
Comment (eivindjahren) [reply tests/integration_tests/scheduler/test_oom.py:12]: Were you thinking of `_ERT_TESTS_ALTERNATE_QUEUE_NAME`? It points to `short` so seems unsuitable. Think it makes sense to just make `_ERT_TESTS_DEFAULT_QUEUE_NAME` be `test` onprem.

File: tests/integration_tests/scheduler/test_oom.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    Path("MEMORY_HOG").write_text("EXECUTABLE memory_hog.sh", encoding="utf-8")
+    Path("memory_hog.sh").write_text(
+        dedent(
+            """#!/bin/sh
+    perl -wE 'my @xs; for (1..2**20) { push @xs, q{a} x 2**20 }; say scalar @xs;'
+    """
+        ),
+        encoding="utf-8",
+    )
+    os.chmod("memory_hog.sh", 0o0755)
```
[the remark this answers is not in the document]
Comment (eivindjahren) [reply tests/integration_tests/scheduler/test_oom.py:51]: Fixed

**Pull request closed without merging**